tests/pytorch/nightly/hf-glue.libsonnet

```diff
@@ -22,46 +22,50 @@ local utils = import "templates/utils.libsonnet";
     git clone https://github.com/huggingface/transformers.git
     cd transformers && pip install .
     git log -1
-    python examples/run_tpu_glue.py \
-      --tensorboard_logdir=$(MODEL_DIR) \
+    python examples/xla_spawn.py \
+      --num_cores 8 \
+      examples/text-classification/run_glue.py \
+      --logging_dir=./tensorboard-metrics \
       --task_name MNLI \
+      --data_dir /datasets/glue/MNLI \
+      --cache_dir ./cache_dir \
       --do_train \
       --do_eval \
-      --data_dir /datasets/glue/MNLI \
+      --num_train_epochs 3 \
       --max_seq_length 128 \
       --learning_rate 3e-5 \
-      --num_train_epochs 3.0 \
       --output_dir MNLI \
       --overwrite_output_dir \
-      --logging_steps 50 \
-      --save_steps 1000 \
-      --num_cores=8 \
-      --only_log_master \
-      --metrics_debug \
+      --logging_steps 100 \
+      --save_steps 3000 \
       --overwrite_cache \
+      --tpu_metrics_debug \
+  |||,
+  local command_copy_metrics = |||
+    gsutil -m cp -r ./tensorboard-metrics/* $(MODEL_DIR)
   |||,
   local bert_base_cased = common.Convergence {
     modelName: "hf-glue-bert-b-c",
     command: utils.scriptCommand(
       |||
-        %(common)s --model_type bert \
-        --model_name_or_path bert-base-cased \
-        --train_batch_size 32 \
-        --eval_batch_size 32
-      ||| % {common: command_common}
+        %(common)s --model_name_or_path bert-base-cased \
+        --per_gpu_train_batch_size 128 \
+        --per_gpu_eval_batch_size 128
+        %(common_copy)s
+      ||| % {common: command_common, common_copy: command_copy_metrics}
     ),
     regressionTestConfig+: {
       alert_for_failed_jobs: false,
```

Comment:
if you think these tests will pass, you can remove all instances of this flag for these tests (it defaults to `true`)

Reply:
Which flag are you referring to?

Reply:
alert_for_failed_jobs